deployment/helm/values.yaml

```diff
@@ -45,15 +45,13 @@ exposedServiceType: NodePort # ClusterIP | NodePort | LoadBalancer
 #   - HUB_WEBAPP_HOST
 #   - HUB_CFSSL_HOST
 #   - ALERT_SERVER_PORT
-# Format: array of strings delimited by commas; each string has the format "KEY:VALUE" e.g. ["HUB_WEBAPP_HOST:localhost", "ALERT_SERVER_PORT:443"]
-environs: []
+environs: {}
 
 # secretEnvirons is a list of environs to add to the Alert's Secret
 # The follwing environs are handled by setting .Values.setEncryptionSecretData:
 #   - ALERT_ENCRYPTION_PASSWORD
 #   - ALERT_ENCRYPTION_GLOBAL_SALT
-# Format: array of strings delimited by commas; each string has the format "KEY:VALUE" e.g. ["KEY:VALUE", "KEY:VALUE"]
-secretEnvirons: []
+secretEnvirons: {}
```

Comment:
same note as above about the example.  also s/follwing/following 🙃

Reply:
done!